securesystemslib/signer/_sigstore_signer.py

```diff
@@ -130,23 +130,22 @@ class SigstoreSigner(Signer):
 
     SCHEME = "sigstore"
 
-    def __init__(self, token: str, public_key: Key):
-        # TODO: Vet public key
-        # - signer eligible for keytype/scheme?
-        # - token matches identity/issuer?
+    def __init__(self, token: Any, public_key: Key):
         self.public_key = public_key
+        # token is of type sigstore.oidc.IdentityToken but the module should be usable
+        # without sigstore so it's not annotated
         self._token = token
 
     @classmethod
     def from_priv_key_uri(
         cls,
         priv_key_uri: str,
         public_key: Key,
-        secrets_handler: Optional[SecretsHandler] = None,
+        _secrets_handler: Optional[SecretsHandler] = None,
```

Comment:
does a falling tree make a sound if no-one is there to hear it?   :thinking: More important I think is that if someone later adds code to this method that uses that argument, tooling would complain (because underscore prefix should mean unused).

I can add a linter skip comment instead if you'd prefer that

Reply:
I slightly lean towards the linter skip. But it's your call. I already approved.